### ISSUE TITLE
Production build doesn't work

### ISSUE THREAD

**Issue by Zaba505** [puku0x/angular-pwa-sample#3]
Hi @puku0x,
I've found this example to be extremely helpful for someone who's just learning how to create a full fledged PWA with Angular. One problem I've ran into is the following:
```
ng run my-app:app-shell:production
Your global Angular CLI version (7.0.3) is greater than your local
version (7.0.1). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".
 10% building 15% building modules 45/45 mod                                                                                       
Date: 2018-10-31T00:54:26.274Z
Hash: 6ba5be968b87eb032015
Time: 20520ms
chunk {main} main.js, main.js.map (main) 115 kB [entry] [rendered]
                                                                                         
Date: 2018-10-31T00:54:43.853Z
Hash: 3818a1a49aabfd61f4f3
Time: 38039ms
chunk {0} runtime.c26e4909e9dc8a449ce4.js (runtime) 2.23 kB [entry] [rendered]
chunk {1} main.06eb0743f1cd35c089e8.js (main) 389 kB [initial] [rendered]
chunk {2} polyfills.c6871e56cb80756a5498.js (polyfills) 38.4 kB [initial] [rendered]
chunk {3} styles.d6b8585595cbf9adcdf8.css (styles) 64.3 kB [initial] [rendered]
chunk {4} 4.6b95f33487f5248bc393.js () 4.09 kB  [rendered]
Angular is running in the development mode. Call enableProdMode() to enable the production mode.
```

As you can see Angular is only building in dev and I was wondering if the is an Angular 7 bug or am I missing something completely? I am able to get production mode by simply deleting the checks in `main.ts` and `main.server.ts`; however, that's makes the development process rather tedious.

**Comment by puku0x** [comment]
Hi @Zaba505,
Thank you for your report. I will check it soon!

**Comment by puku0x** [comment]
@Zaba505
I found `"serverTarget": "my-app:server:production"` was missed in the production build setting.

```
"app-shell": {
  "builder": "@angular-devkit/build-angular:app-shell",
  "options": {
    "browserTarget": "my-app:build",
    "serverTarget": "my-app:server",
    "route": "shell"
  },
  "configurations": {
    "production": {
      "browserTarget": "my-app:build:production",
      "serverTarget": "my-app:server:production"
    }
  }
}
```

I think this will fix the problem!

**Comment by Zaba505** [comment]
@puku0x 
Thanks, that worked perfectly! I'm guessing that must be something missing in the Angular `appShell` or `universal` schematics. Would you like me to keep this open until that fix is added?

**Comment by puku0x** [comment]
@Zaba505 
Yeah, sure. I will send a report to Angular team later.

**Comment by puku0x** [comment]
The patch was merged but it seems it will be fixed after CLI 7.1 released👍

**Comment by puku0x** [comment]
@Zaba505 
Yay! Angular CLI v7.1.0 was released and the above patch was available now!
